app/decorators/decidim/proposals/import/proposal_creator_decorator.rb

```diff
@@ -10,6 +10,19 @@ def finish_without_notif!
         end
         resource
       end
+
+      def produce
```

Comment:
this method name comes from Decidim?

Reply:
Hi @laurajaime ,
Yes, I want to replace the method produce from proposal_creator to add the possibility to manage different types of author depending on the import file data. By default it only takes the current user as author I think.